files/embed.go

```diff
@@ -0,0 +1,10 @@
+package files
+
+import (
+	// Embed installer template
+	_ "embed"
+)
+
+// InstallerTemplateSource is a string containing the installer template
+//go:embed "install.sh.tmpl"
+var InstallerTemplateSource string
```

Comment:
Put this embed the top level, there's no need to have an intermediate .go file here.